app/todo/api.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import APIRouter, Depends
+
+from app.depends.auth import get_current_user
+from app.depends.service import get_todo_service
+from app.todo.models import (DeleteTodo, DeleteTodoInfo,
+                             GetTodoInfo, GetTodoInfoList, InsertTodoInfo)
+from app.todo.service import BaseTodoService
+
+app_router = APIRouter()
+
+
+@app_router.get("/todoinfo", response_model=GetTodoInfoList)
+async def getTodoInfoList(
+    user_id: int = Depends(get_current_user), todo_svc: BaseTodoService = Depends(get_todo_service),
+):
+    todoinfo = await todo_svc.getTodoInfoList(user_id)
+    return GetTodoInfoList(data=todoinfo)
+
+
+@app_router.post("/regtodo", response_model=InsertTodoInfo, status_code=201)
+async def insertTodo(
+    context: str, user_id: int = Depends(get_current_user), todo_svc: BaseTodoService = Depends(get_todo_service),
+):
+    todoinfo = await todo_svc.insetTodoInfo(user_id, context)
+    return GetTodoInfo(data=todoinfo)
+
+
+@app_router.post("/deltodo", response_model=DeleteTodoInfo)

```

Comment:
```suggestion
@app_router.delete("/todo", response_model=DeleteTodoInfo)

```
위와 같은 논리로, 이는 `DELETE /todo`로 정의해도 충분해 보입니다!

Reply:
변경완료했습니다!
10fcdca476d40ccf1fd8c9d6a29a30c9f744a525